app/services/govuk_emails/monitor.rb

```diff
@@ -9,22 +14,67 @@ def initialize(scheduled_mail_id)
     end
 
     def call
-      status = GovukEmails::Email.new(@scheduled_mail.govuk_message_id).status
-      @scheduled_mail.update!(status:)
-      send_undeliverable_alerts if status.in?(ScheduledMailing::FAILURE_STATUSES)
+      scheduled_mail.update!(status: response.status)
+
+      send_undeliverable_alerts if scheduled_mail.status.in?(ScheduledMailing::FAILURE_STATUSES)
     end
 
   private
 
+    def response
+      @response ||= GovukEmails::Email.new(scheduled_mail.govuk_message_id)
+    end
+
     def send_undeliverable_alerts
-      return unless HostEnv.production?
+      # return unless HostEnv.production?
 
-      AlertManager.capture_message("Unable to deliver mail to #{@scheduled_mail.addressee} - ScheduledMailing record #{@scheduled_mail.id}")
+      AlertManager.capture_message("Unable to deliver mail to #{scheduled_mail.addressee} - ScheduledMailing record #{scheduled_mail.id}")
+
+      send_undeliverable_to_apply_team!
+      send_undeliverable_citizen_start_email_to_provider!
+    end
+
+    def send_undeliverable_to_apply_team!
```

Comment:
This could be 'undeliverable_alert' rather than just 'undeliverable' to make it slightly clearer?